src/Build/NuGet.Build.Packaging.Tasks/NuGet.Build.Packaging.targets

```diff
@@ -63,6 +63,11 @@ Copyright (c) .NET Foundation. All rights reserved.
 			<IsPackable>$(IsPackable)</IsPackable>
 			<IsNuGetized>$(IsNuGetized)</IsNuGetized>
 		</TargetPathWithTargetPlatformMoniker>
+		<!-- Design Time is annoying and uses a different output item group to return this. Also adust that -->
+		<_MainProjectOutput>
```

Comment:
Not really, I think, since it's a VC++ targets specific issue, rather than an MSBuild one.